crates/miden-lib/asm/miden/contracts/faucets/mod.masm

```diff
@@ -17,8 +17,10 @@ const.ERR_BASIC_FUNGIBLE_BURN_WRONG_NUMBER_OF_ASSETS="burn requires exactly 1 no
 # CONSTANTS
 # =================================================================================================
 
-# The slot in this component's storage layout where the metadata is stored.
-const.METADATA_SLOT=0
+# The slot in this component's storage where the metadata is stored.
+# TODO(named_slots): Unify slot name or make distribute take name ID as a parameter to allow for
+# different slot names per fungible faucet?
+const.METADATA_SLOT=word("miden::basic_fungible_faucet::metadata")
```

Comment:
My current thinking is that it may be better to unify slot naming (though, I'd use something like `miden::standards::faucets::basic_fungible_faucet::metadata`).

One thing we should consider is how we handle components that want to use the same slot. For now, I believe, we don't allow this (though, I wonder what kind of error messages the user would get if they try to add such conflicting components to an account). In the future, maybe there are some strategies for "merging" such components (assuming there are real use cases for this).